ERROR    ProductMaterials:product_materials.py:97 请求发生错误: HTTPConnectionPool(host='192.168.0.135', port=8282): Max retries exceeded with url: /MaterialInfoApi/GetMaterialInfoAutoQueryDatas (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000026366AA99D0>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))，请求 URL: http://192.168.0.135:8282/MaterialInfoApi/GetMaterialInfoAutoQueryDatas，请求头: {'authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJVc2VySWQiOiIxNTAiLCJVc2VyQ29kZSI6IkRlbW9BZG1pbiIsIlVzZXJOYW1lIjoiRGVtb0FkbWluIiwiVXNlclR5cGUiOiIxIiwiQ29tcGFueUNvZGUiOiIwMDAwMCIsIkZhY3RvcnlDb2RlIjoiMDAwMDAuMDAwMDEiLCJJcCI6IjE5Mi4xNjguODIuMjA0IiwiT3JnYW5pemF0aW9uU3RydWN0dXJlQ29kZSI6IjAwMDAwLjAwMDAxLjAwMDAzIiwiZXhwIjoxNzUyNDU5OTE1LCJpc3MiOiJSaC5NZXMiLCJhdWQiOiJSaC5NZXMuU2VydmljZXMifQ.MN4rWGzqN8MvEce6QggdHVfK5S1n54GrnS07ShqJNi0'}，请求体: {'MaterialCode': 'Automation01'}
ERROR    mainprocess_tests:test_mainprocess.py:129 查询物料失败：getMaterialInfoAutoQueryDatas接口返回None，请检查接口或依赖服务是否正常
assert None is not None